src/main/java/com/exasol/containers/ExasolPurger.java

```diff
@@ -0,0 +1,96 @@
+package com.exasol.containers;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class purges all objects from an Exasol database.
+ */
+// [impl->dsn~purging~1]
+public class ExasolPurger {
```

Comment:
Exasol Purger (e.g, The Purge movie) sounds scary :) 

Suggestion: use ExasolDatabaseCleaner

Reply:
On the second thought, maybe this whole class functionality should exist in [test-db-builder-java](https://github.com/exasol/test-db-builder-java)

Reply:
I think it fits better here. test-db-builder has a similar functionality to clean up the objects it created. This one is however different since it deletes just everything.

Apart from that test-db-builder uses testcontiners. For that reason, we can't use it here since it would lead to a cyclic dependency.